students/philip_behrend/lesson09/assignment/pngdiscover.py

```diff
@@ -9,8 +9,34 @@ def parse_args():
     parser.add_argument('-i', '--input', help="Input directory", required=True)
     return parser.parse_args()
 
-def find_png(parent_dir, png_list):
+def find_png(parent_dir, output_list):
     """ Find all PNG files within parent directory and return list of file names """
+    os.chdir(parent_dir)
```

Comment:
You don't need to change directories within your script, just provide the right path for _os.listdir_ to work with.